aspnet/publishing/azure-continuous-deployment.rst

```diff
@@ -1,8 +1,249 @@
+
+	.. note:: Your password will not be visible as you enter it.
+	
+This command will start the process of pushing your local project files to Azure. The output from the above command ends with a message that deployment was successful.
+	
+|	remote: Finished successfully.
+|	remote: Running post deployment command(s)...
+|	remote: Deployment successful.
+|	To https://username@samplewebappdemo01.scm.azurewebsites.net:443/SampleWebAppDemo01.git
+|	* [new branch]      master -> master
+|	Branch master set up to track remote branch master from Azure-SampleApp.
+|
+
+.. note:: If you need to collaborate on a project, you should consider pushing to `GitHub <https://github.com>`_ in between pushing to Azure.
+
+Verify the Active Deployment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You can verify that you successfully transferred the web app from your local environment to Azure. You'll see the listed successful deployment.
+
+1. In the `Azure Portal <https://portal.azure.com>`_, select your web app. Then, select **Settings** > **Continuous deployment**. 
+
+	.. image:: azure-continuous-deployment/_static/13-verify-deployment.png
+
+Run the app in Azure
+--------------------
+Now that you have deployed your web app to Azure, you can run the app. 
+
+This can be done in two ways:
+
+* In the Azure Portal, locate the web app blade for your web app, and click **Browse** to view your app in your default browser.
+* Open a browser and enter the URL for your web app. For example:
+
+	``http://SampleWebAppDemo.azurewebsites.net``
+
+Update your web app and republish
+---------------------------------
+After you make changes to your local code, you can republish. 
+
+1. In **Solution Explorer** of Visual Studio, open the *Startup.cs* file.
+
+2. In the ``Configure`` method, modify the ``Response.WriteAsync`` method so that it appears as follows:
+
+.. code-block:: aspx-cs
+
+	await context.Response.WriteAsync("Hello World! Deploy to Azure.");
+	
+3. Save changes to *Startup.cs*.
+
+4. In **Solution Explorer**, right-click **Solution 'SampleWebAppDemo'** and select **Commit**. The **Team Explorer** will be displayed. 
+
+5. Enter a commit message, such as:
+
+.. code-block:: none
+
+	Update #2
+	
+6. Press the **Commit** button to commit the project changes.
+
+7. Select **Home** > **Sync** > **Actions** > **Push**. 
+
+	.. note:: As an alternative, you can push your changes from the **Command Window** by opening the **Command Window**, changing to your project directory, and entering a git command. For example:
+	
+		``git push -u Azure-SampleApp master``
+
+View the updated web app in Azure
+---------------------------------
+
+View your updated web app by selecting **Browse** from the web app blade in the Azure Portal or by opening a browser and entering the URL for your web app. For example:
+
+	``http://SampleWebAppDemo.azurewebsites.net``
+
+Additional Resources
+--------------------
+- ASP.NET 5 :doc:`/publishing/index`
```

Comment:
This is going to result in some odd formatting where just the link is italicized. Just override the entire tile:
```
:doc:`ASP.NET 5 Publishing </publishing/index>`
```